# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../conntrack-tools/0001-include-sys-types.h.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

include sys/types.h for getting u_int*_t types

--- ./src/filter.c.orig	2013-09-01 02:08:56.263022781 +0200
+++ ./src/filter.c	2013-09-01 02:10:29.529194635 +0200
@@ -17,6 +17,8 @@
  * Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
  */
 
+#include <sys/types.h>
+
 #include "filter.h"
 #include "bitops.h"
 #include "jhash.h"
--- ./src/mcast.c.orig	2013-09-01 02:14:45.331137392 +0200
+++ ./src/mcast.c	2013-09-01 02:15:15.979832693 +0200
@@ -18,6 +18,8 @@
  * Description: multicast socket library
  */
 
+#include <sys/types.h>
+
 #include "mcast.h"
 
 #include <stdio.h>
--- ./src/channel_mcast.c.orig	2013-09-01 02:16:10.133076145 +0200
+++ ./src/channel_mcast.c	2013-09-01 02:16:35.301660985 +0200
@@ -7,6 +7,7 @@
  * (at your option) any later version.
  */
 
+#include <sys/types.h>
 #include <stdlib.h>
 #include <libnfnetlink/libnfnetlink.h>
 
--- ./src/channel_tcp.c.orig	2013-09-01 02:17:42.787237433 +0200
+++ ./src/channel_tcp.c	2013-09-01 02:18:08.731846232 +0200
@@ -9,6 +9,7 @@
  * TCP support has been sponsored by 6WIND <www.6wind.com>.
  */
 
+#include <sys/types.h>
 #include <stdlib.h>
 #include <libnfnetlink/libnfnetlink.h>
 
--- ./src/channel_udp.c.orig	2013-09-01 02:18:18.588072866 +0200
+++ ./src/channel_udp.c	2013-09-01 02:18:36.172490868 +0200
@@ -7,6 +7,7 @@
  * (at your option) any later version.
  */
 
+#include <sys/types.h>
 #include <stdlib.h>
 #include <libnfnetlink/libnfnetlink.h>
 
